docs/guides/get-qpu-information.ipynb

```diff
@@ -254,6 +254,47 @@
     "For a full list of attributes, see the [`IBMBackend` API documentation](/docs/api/qiskit-ibm-runtime/ibm-backend)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Native gates\n",
+    "\n",
+    "Each [processor family](/docs/guides/processor-types) has a native gate set. By default, the QPUs in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set.\n",
+    "\n",
+    "You can view the native gates and operations for a QPU either [with Qiskit](#native-gates-with-qiskit) or on the IBM Quantum&reg; Platform [Compute resources page](#native-gates-on-platform).\n",
+    "\n",
+    "<Admonition type=\"note\" title=\"Basis gates\">\n",
+    "The terms native gates and basis gates are often used interchangeably. However, you can specify a different set of basis gates to use, while the native gate set never changes. For information about changing the basis gates, see the [Represent quantum computers](/docs/guides/represent-quantum-computers#basis-gates) topic.\n",
+    "</Admonition>\n",
+    "\n",
+    "<span id=\"native-gates-with-qiskit\"></span>\n",
+    "### With Qiskit\n",
+    "\n",
+    "```python\n",
+    "\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "service = QiskitRuntimeService(channel=\"ibm_quantum_platform\")\n",
+    "\n",
+    "for backend in service.backends():\n",
+    "    config = backend.configuration()\n",
+    "    if \"simulator\" in config.backend_name:\n",
+    "        continue\n",
+    "    print(f\"Backend: {config.backend_name}\")\n",
+    "    print(f\"    Processor type: {config.processor_type}\")\n",
+    "    print(f\"    Supported instructions:\")\n",
+    "    for instruction in config.supported_instructions:\n",
+    "        print(f\"        {instruction}\")\n",
+    "    print()\n",
+    "```\n",
+    "\n",
+    "<span id=\"native-gates-on-platform\"></span>\n",
+    "### On IBM Quantum Platform\n",
```

Comment:
@jyu00 Should I skip bringing over the info on how to find supported gates on the Platform, since this page is specific to Qiskit? I guess I could move this piece to the [QPU information](https://quantum.cloud.ibm.com/docs/en/guides/qpu-information) page?.... thoughts?

Reply:
[Here's the preview](https://qiskit.github.io/documentation/pr-3613/docs/guides/get-qpu-information#on-ibm-quantum-platform) for more context